ansible/roles/openshift_sso_app_zabbix/README.md

```diff
@@ -18,6 +18,12 @@ Role Variables
 Dependencies
 ------------
 
+- `ossoz_zbx_server`: The name of the Zabbix server
+- `ossoz_zbx_user`: The user with which to authenticate to the Zabbix server
+- `ossoz_zbx_password`: The password of the user
+- `ossoz_masters`: The list of masters in the cluster
+- `ossoz_clusterid`: The id of the cluster
+
```

Comment:
sorry, GH comment ambiguity strikes again. I meant that these should go under "Role Variables"